### PR TITLE
Fastdns

### PR DESCRIPTION
**Pull request by jeffersongirao**

### CODE REVIEW

File: fastdns/fastdns.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+// len(0): use credentials from environment
+// len(4): credentials[0] = Akamai Host
+//         credentials[1] = Client Token
+//         credentials[3] = Client Secret
+//         credentials[4] = Access Token
+func NewDNSProvider(credentials ...string) (caddytls.ChallengeProvider, error) {
+	switch len(credentials) {
+	case 0:
+		return fastdns.NewDNSProvider()
+	case 2:
```
Comment (mholt) [inline fastdns/fastdns.go]: Oops, this should be `case 4` :)

**Review by mholt — CHANGES_REQUESTED** [review]
Thanks! And thanks for fixing namedotcom. Just one little tweak and then I will merge. :)

File: fastdns/fastdns.go (hunk 1 above)
Comment (jeffersongirao) [reply fastdns/fastdns.go]: Ooops, brainfart :D It is fixed now I think.

**Review by mholt — APPROVED** [review]

**Comment by mholt** [comment]
Awesome, thank you!

**Pull request merged**